packages/cursorless-neovim/scripts/populate-dist.sh

```diff
@@ -2,6 +2,9 @@
 set -euo pipefail
 
 echo "Populating dist directory..."
+if [ ! -e "${CURSORLESS_REPO_ROOT-nonexistent}" ]; then
```

Comment:
Is `-nonexistent` a typo?

Reply:
no, it's intentional. its a default for if the env variable is unset.

you can think of it as how python has foo=None as default func arguments, so caller doesn't have to pass foo but can still be used inside the function. In this case the script uses `set -euo pipefail` where the `u` means you can't use undefined variables (which is generally good to have). But then since `CURSORLESS_REPO_ROOT` isn't explicitly set inside the script, it would be considered an undefined variable and fail. But we know the variable will have been set outside the script, so we just use this syntax to say if it wasn't set initialized it to the string "nonexistent" which is a filename that won't exist and so will enter the if condition and complain gracefully.